pkg/bundle/manager.go

```diff
@@ -66,6 +80,11 @@ func (m bundleManager) ProcessBundle(_ context.Context, newBundle *api.PackageBu
 	return false, nil
 }
 
+func (m *bundleManager) isCompatibleWith(bundle *api.PackageBundle) bool {
+	currentVersion := m.config.BuildInfo.Version
+	return currentVersion == config.DEVELOPMENT || semver.Compare(currentVersion, bundle.Spec.MinVersion) >= 0
```

Comment:
seems like the semver comparison belongs in the config object not to mention the DEVELOPMENT thing. The bundle manager should be blissfully ignorant.

Reply:
Determining if the bundle is compatible by comparing the minimum required version is part of the business logic of the bundle controller. The controller decides if it admits the bundle or not based on requirements that should not be part of a generic config package that holds application configuration.